static/images/thing-details-link.svg

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You probably could have just used the PNG from the "Things" item in the main menu. Let me export an SVG that matches that a bit better...

Reply:
I've switched to the PNG, thanks for pointing out that it exists!